### ISSUE TITLE
Facebook login

### ISSUE THREAD

**Issue by timdeng2324** [tomasmcm/SpotiWeb#10]
I'm having trouble using Facebook as a login option when opening the app. After clicking the "log in with Facebook", a new window pops up prompting me to log in. I perform the necessary steps, including my password, two-factor authentication, clicking save browser, and confirming I am logging in from my location. Then, the pop-up screen does nothing and just turns black. If I close it, I am not logged in. I have also tried typing my Facebook email and password directly into the login, but it doesn't work the same way it does in the official Spotify app.

**Comment by tomasmcm** [comment]
@timdeng2324 yeah the facebook login is a bit strange... I've been told that if you login with facebook you have to close the app and open it again for it to work.
